VERSION.txt

```diff
@@ -137,7 +137,7 @@ jetty-9.4.35.v20201120 - 20 November 2020
  + 5562 ArrayTernaryTrie consumes too much memory
  + 5575 Add SEARCH as a known HttpMethod
  + 5605 CVE-2020-27218 java.io.IOException: unconsumed input during http
-   request parsing
+   request parsing - Resolves CVE-2020-27218
```

Comment:
The CVE identifier is listed at the start of this entry (the line above this change)